conformance/reports/v1.0.0/azure-application-gateway-for-containers/README.md

```diff
@@ -0,0 +1,13 @@
+# Azure Application Gateway for Containers
+
+[Application Gateway for Containers][azure-application-gateway-for-containers] is a managed application (layer 7) load balancing solution, providing dynamic traffic management capabilities for workloads running in a Kubernetes cluster in Azure. Follow the [quickstart guide][azure-application-gateway-for-containers-quickstart-controller] to deploy the ALB controller and get started with Gateway API.
+
+## Table of Contents
+
+|API channel|Implementation version|Mode|Report|
+|-----------|----------------------|----|------|
+|x|[v1.0.0](https://learn.microsoft.com/azure/application-gateway/for-containers/alb-controller-release-notes#latest-release-recommended)|x|[link](./v1.0.0-report.yaml)|
+
+
```

Comment:
We've started to require reproduction steps for all new conformance reports, do you mind filling out something like this?

https://github.com/kubernetes-sigs/gateway-api/tree/main/conformance/reports/v1.0.0/kong-kubernetes-ingress-controller#prerequisites

It may also be useful to look at [AKS upstream conformance](https://github.com/cncf/k8s-conformance/tree/master/v1.27/aks) here because this will probably be the first conformance report that requires spinning up a cluster on a specific cloud provider.